types/coin.go

```diff
@@ -842,46 +841,17 @@ func (coins Coins) Sort() Coins {
 	return coins
 }
 
-//-----------------------------------------------------------------------------
-// Parsing
-
-var (
-	// Denominations can be 3 ~ 128 characters long and support letters, followed by either
-	// a letter, a number or a separator ('/', ':', '.', '_' or '-').
-	reDnmString = `[a-zA-Z][a-zA-Z0-9/:._-]{2,127}`
-	reDecAmt    = `[[:digit:]]+(?:\.[[:digit:]]+)?|\.[[:digit:]]+`
-	reSpc       = `[[:space:]]*`
```

Comment:
Should we keep this in some form as docs for what were matching?

Reply:
Done in https://github.com/cosmos/cosmos-sdk/pull/19511/commits/fac684ef1275c03163700921f2972f8049f2ad2a!